rollouts/config/default/kustomization.yaml

```diff
@@ -27,7 +27,7 @@ namePrefix: rollouts-
 #  someName: someValue
 
 bases:
-- ../crd
+# - ../crd
```

Comment:
Maybe just delete the kustomize setup that comes with Kubebuilder. It is kinda complicated, so we can just remove it if we don't need it. The controllers in Porch has a very simple configuration.

Reply:
I am keeping them around till our APIs becomes stable.  I also found them to be complicated, but just using them for generation purpose for now. Will remove them eventually.